.github/workflows/lint.yml

```diff
@@ -0,0 +1,58 @@
+name: Lint
+on:
+  push:
+    paths:
+      - "**.go"
+      - go.mod
+      - go.sum
+  pull_request:
+    paths:
+      - "**.go"
+      - go.mod
+      - go.sum
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
```

Comment:
This include both golangci and fmt in one job ?

Reply:
yes